src/fred/cli/releases.py

```diff
@@ -0,0 +1,135 @@
+#!/usr/bin/env python3
+"""
+FRED CLI - Releases Namespace.
+"""
+import json
+
+import click
+
+from .. import BaseFredAPIError
+from .._util import generate_api_kwargs
+
+
+__all__ = [
+    "releases",
+]
+
+
+@click.group()
+@click.pass_context
+def releases(ctx):
+    """
+    Releases CLI Namespace.
+    """
+    pass
+
+
+@releases.command()
+@click.argument("args", nargs=-1)
+@click.pass_context
+def get_releases(ctx, args: tuple):
+    """Get releases."""
+    try:
+        click.echo(json.dumps(ctx.obj["client"].get_releases(**generate_api_kwargs(args)), indent=4))
+    except (ValueError, BaseFredAPIError) as e:
+        click.echo(click.style(e, fg="red"))
+
+
+@releases.command()
+@click.argument("args", nargs=-1)
+@click.pass_context
+def get_releases_dates(ctx, args: tuple):
+    """Get a category."""
```

Comment:
Fix docstring here

Reply:
Additionally, there are a few other incorrect docstrings when copying the scaffolding code, please fix them, they affect the help string output of the CLI